source/renderer/app/components/staking/DelegationInfo.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
+import moment from 'moment';
+import styles from './DelegationInfo.scss';
+
+type Props = {
+  heading: string,
+  info: string,
+  timeLeftDesc: string,
+  timeLeft: number,
+  buttonLabel: string,
+};
+
+@observer
+export default class DelegationInfo extends Component<Props> {
+  render() {
+    const { heading, info, timeLeftDesc, timeLeft, buttonLabel } = this.props;
+    const duration = moment.duration(timeLeft);
+    const days = duration.days();
+    const hours = duration.hours();
+    const minutes = duration.minutes();
+    const timeLeftString = `${days} days ${hours} hours ${minutes} minutes`;
```

Comment:
@yakovkaravelov we need to translate this. Please see how it is done here: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/components/loading/SystemTimeErrorOverlay.js#L104-L128